gson/src/test/java/com/google/gson/functional/ConcurrencyTest.java

```diff
@@ -15,62 +15,43 @@
  */
 package com.google.gson.functional;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.google.gson.Gson;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.atomic.AtomicBoolean;
-
-import junit.framework.TestCase;
-
-import com.google.gson.Gson;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 /**
  * Tests for ensuring Gson thread-safety.
  * 
  * @author Inderjeet Singh
  * @author Joel Leitch
  */
-public class ConcurrencyTest extends TestCase {
+class ConcurrencyTest {
   private Gson gson;
 
-  @Override
-  protected void setUp() throws Exception {
-    super.setUp();
+  @BeforeEach
+  void setUp() throws Exception {
     gson = new Gson();
   }
 
   /**
    * Source-code based on
    * http://groups.google.com/group/google-gson/browse_thread/thread/563bb51ee2495081
    */
-  public void testSingleThreadSerialization() { 

```

Comment:
Removed these two test methods because they were not performing any assertion. And there are (probably) enough tests already performing basic reflection-based serialization and deserialization.